pkg/test/framework/features/features.yaml

```diff
@@ -18,7 +18,18 @@ features:
   # features that relate to measuring or exploring your service mesh or the services which comprise it.
   observability:
     telemetry:
-    tracing:
+      stats:
```

Comment:
curious: how do we explore dimensions of features (CNI, VM, Multicluster)?  

and where do things like "customizable metrics" and "remote extensions" stuff plugin on this list?

Reply:
How about `observability.telemetry.stats.env.{CNI, VM, Multicluster}`?

I added customizable metrics into feature list and make request classification test cover both request-classfication and metric customization.

For extension related stuff, there is actually a section `extensibility` at the bottom of the list, remote load would be a good fit there.

Reply:
That is a good question.  `env` is a cross cutting concern. 
I expect to ask a question like "Are all telemetry features being covered in multi cluster testing" 
There is multi cluster / multi network, multi cluster flatnetwork?Should we try to encode this info or provide some other way.
@douglas-reid @therealmitchconnors

Reply:
Sorry I missed this thread.  Environment is measured separately from the feature label.  Currently, we track environments like MultiCluster and VM (this is still in progress).  Can you point me to an example of a CNI test, and I will investigate how to add that to the environment dimension?